config/custom_rules.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: "test"
+description: "if query from etl, route to etl group"
+condition: "request.getHeader(\"X-Trino-Catalog\") == \"uptycs\" && (request.getHeader(\"X-Trino-User\") == \"uptycs\" )"
```

Comment:
@ryadav-uptycs since this is public facing org (uptycslabs) let's keep these rules probably in edna or in kubernetes config of presto-gateway

Reply:
this config and rules are part of uptycs-kubernetes repo only . This folder is not required anymore I will delete it